packages/tocco-ui/src/FormattedValue/typeFormatters/TimeFormatter.js

```diff
@@ -27,12 +28,7 @@ const TimeFormatter = props => {
 
 TimeFormatter.propTypes = {
   intl: intlShape.isRequired,
-  value: PropTypes.shape({
-    hourOfDay: PropTypes.number,
-    minuteOfHour: PropTypes.number,
-    secondOfMinute: PropTypes.number,
-    millisOfSecond: PropTypes.number
-  }).isRequired
+  value: PropTypes.string
```

Comment:
maybe validate with regex (also better just for documentation purposes IMO)

e.g.:

```
value: (props, propName, componentName) {
    if (!/^\d{2}:\d{2}(:\d{2}(:\d{3})?)?$/.test(props[propName])) {
      return new Error('Invalid prop `' + propName + '` supplied to `' + componentName + '`')
    }
  }
```